gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleWrapperPatcher.java

```diff
@@ -69,6 +90,59 @@ private static void patchGradlewContent(
         write(patchedGradlewScript.getAsFile().get().toPath(), getNewGradlewWithPatchContent(linesNoPatch));
     }
 
+    private static void patchGradlewJar(
+            Path buildDir, RegularFileProperty originalGradleWrapperJar, RegularFileProperty patchedGradleWrapperJar) {
+        JarResources.extractJar(originalGradleWrapperJar.getAsFile().get(), buildDir);
+
+        OrigGradleWrapperCreator.create(buildDir);
+        String[] classPaths = System.getProperty("java.class.path").split(File.pathSeparator);
```

Comment:
Reading this from the classpath feels a bit funky as we have then have our `org.gradle.wrapper.GradleWrapperMain` in the gradle plugins classpath even though we don't use it. Shouldn't this already be in the repo somewhere? Can't remember how/where we create the `gradle/gradle-jdks-setup.jar`.  That or I think it would technically be better to either:

1) Include the gradle-wrapper-setup.jar in the resources of the this jar, so we can still read it easily (probably the easiest way).
2) Use Gradle to download the jar by making a `Configuration`, adding the dep with the correct version and resolving it.